antsibull/data/docsite/plugin.rst.j2

```diff
@@ -60,6 +60,9 @@
 
 {% if doc['version_added'] is still_relevant -%}
 .. versionadded:: @{ doc['version_added'] }@
+{%   if 'version_added_collection' in doc and doc['version_added_collection'] != collection_name %}
+                  of @{ doc['version_added_collection'] | rst_ify }@
```

Comment:
I think the way we've written the schema, you don't have to check `version_added_collection in doc` anymore.  (And that's how I like to write the schema whenever possible.  The schema creates a default value for the field and then you don't have to check whether the field exists, you can skip directly to checking whether the field contains something that you need to deal with).

Reply:
I eventually noticed that and changed some, but apparently not all places...